js/danoni_main.js

```
@@ -12027,19 +12027,30 @@ const getArrowSettings = () => {
+			];
+		});
+		const _styleTransDf = structuredClone(Object.values(_styleTrans));
+
 		if (g_stateObj.swapping === `Mirror`) {
-			_style.map(_group => _group.reverse());
+			_styleTrans.map(_group => _group.reverse());
 
 		} else if (g_stateObj.swapping === `X-Mirror`) {
 			// X-Mirrorの場合、グループの内側だけ入れ替える
-			_style.forEach((group, i) => g_settings.swapPattern.forEach(val => swapGroupNums(_style, group, i, val)));
+			_styleTrans.forEach((group, i) => g_settings.swapPattern.forEach(val => swapGroupNums(_styleTrans, group, i, val)));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)